src/ImageSequencer.js

```diff
@@ -292,6 +299,30 @@ ImageSequencer = function ImageSequencer(options) {
     });
   }
 
+  function loadNewModule(name,options){
+
+    if(!options) {
+      return this;
+    }
+    else if(Array.isArray(options)) {
+      // contains the array of module and info
+      this.module[name] = options;
+
+    } else  if(options.func && options.info) {
+      // passed in options object
+      this.modules[name] = [
+        options.func, options.info
+      ]
```

Comment:
semicolon -- although these are being left out more and more, unless we really do it throughout the whole library, we should be consistent. Thanks!